### PR TITLE
First draft of the decision-making process section

### PR DESCRIPTION
**Pull request by milancurcic**
Please review and let me know if I missed anything.

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -26,6 +26,50 @@ Current administrators are
...
+  The purpose of the 80% requirement is to require a significant majority
+  agreement on a decision, while preventing any single member from blocking the
+  group from moving forward on that decision.
+* The administrators will require 51% agreement on all other decisions.
+* The administrator team will have at least five members. If an administrator
+  leaves the team for any reason, in which event the number of administrators
+  would become less than five, the remaining administrators will make their
+  best effort to replace the departing administrator.
+* No single administrator will act on behalf of Fortran-lang without first
+  discussing it with the administrators team.
```
Comment (certik) [inline GOVERNANCE.md:71]: * Each administrator has a veto power and can veto a decision. This power should be only used as a last resort.

Comment (gnikit) [reply GOVERNANCE.md:71]: That seems reasonable but how do we resolve the cases where +80% agreement has been reached but a veto has also been issued?

**Comment by milancurcic** [comment]
I'll work on this tonight.

**Review by certik — APPROVED** [review]
I think this is fine. Thanks Milan for writing this up.

**Review by awvwgk — APPROVED** [review]
Looks good to me.

**Comment by gnikit** [comment]
Looks great (a bit late to the party)! I think we need to update the admin list and github team.

**Comment by milancurcic** [comment]
Thanks, @gnikit. About the admin list, do you mean #4?

**Comment by gnikit** [comment]
> Thanks, @gnikit. About the admin list, do you mean #4?

Yeap, way ahead of me, Thanks!

**Pull request merged**